core/src/luigi-config/utils/params-version.js

```diff
@@ -0,0 +1,23 @@
+import { major, satisfies, coerce } from 'semver';
+
+const PARAMS_VERSION = '1.0'; // make sure to sync it in core-ui
```

Comment:
can we move init-params-related constants to one file? (stuff like DEFAULT_MODULES etc that we have in shared)